app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,19 @@
+class ReservationCalendar < SimpleDelegator
+
+  def initialize(reservation)
+    super(reservation)
+  end
+
+  def as_ical
+    cal = Icalendar::Calendar.new
+    cal.event do |e|
+      e.dtstart = Icalendar::Values::DateTime.new(reserve_start_at)
+      e.dtend = Icalendar::Values::DateTime.new(reserve_end_at)
+      e.summary = "Reservation for #{product.name}"
+      e.description = "Reservation for #{product.name}"
```

Comment:
I think we should flesh out the description (I'm assuming summary is sort of the "subject" and the desciption is the "body"/"notes"). We should include the full facility name and the order number. If we can do HTML in there, I would link to the order receipt. Or is there a URL field?

Maybe add the facility name as the location?